docs/HTTP-batchsource.md

```diff
@@ -0,0 +1,206 @@
+# HTTP Batch Source
+
+Description
+-----------
+This plugin reads data from HTTP/HTTPS pages.
+Using paginated API to retrieve data is allowed.
+Data in JSON, XML, CSV, TSV, TEXT and BLOB formats is supported.
+
+Properties
+----------
+
+**Reference Name:** Name used to uniquely identify this source for lineage, annotating metadata, etc.
+
+### General
+
+**URL:** Url to fetch to the first page.
+The url must start with a protocol (e.g. http://).
+
+Supports {pagination.index} placeholder, neccessary for pagination
+type "Increment an index". For more information read pagination section.
+
+**HTTP Method:** Request HTTP method.
+
+**Headers:** Request HTTP headers for the.
+
+**Request body:** A body of the request.
+
+### Format
+
+**Format:** Format of pages. Used to determine how to convert pages into output records. Possible values are:<br>
+JSON - retrieves all records from given location (json path)
+and transforms them into records according to the mapping.<br>
+XML - retrieves all records from given location (XPath)
+and transforms them into records according to the mapping.<br>
+TSV/CSV - transforms TSV/CSV pages into records. Page columns are mapped to record fields in the order they are
+listed in schema.<br>
+Text - transforms a single line of text into a single record with a string field "body" containing the result.<br>
+BLOB - transforms a single page into a single record with a byte array field "body" containing the result.
+
+**JSON/XML Result Path:** Path to the results. When XML is used this is an XPath, when json is used this is a JSON path.
+JSON path examples: "/bookstore/books", "/feed/entries/items". JSON path should either point to an array in a json
```

Comment:
Include an XPath example?

Reply:
Along with example paths, we should include example responses so it is more clear to the user.

Reply:
done